applications/matter_bridge/doc/adding_bridged_matter_device.rst

```diff
@@ -0,0 +1,397 @@
+               DeviceLayer::PlatformMgr().ScheduleWork(NotifyAttributeChange, reinterpret_cast<intptr_t>(provider));
+            }
+
+            void SimulatedPressureSensorDataProvider::NotifyAttributeChange(intptr_t context)
+            {
+               SimulatedPressureSensorDataProvider *provider = reinterpret_cast<SimulatedPressureSensorDataProvider *>(context);
+               provider->NotifyUpdateState(Clusters::PressureMeasurement::Id,
+                           Clusters::PressureMeasurement::Attributes::MeasuredValue::Id,
+                           &provider->mPressure, sizeof(provider->mPressure));
+            }
+
+   #. Implement the body of :c:func:`NotifyUpdateState` method that shall be called after every data change related to the ``Pressure Sensor``.
+      It is used to inform the ``Bridge Manager`` and Matter Data Model that attribute value should be updated.
+      To make the method invoke appropriate callback, edit :file:`simulated_pressure_sensor_data_provider.cpp` file as follows:
+
+      .. code-block:: C++
+
+         void SimulatedPressureSensorDataProvider::NotifyUpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId,
+                                    void *data, size_t dataSize)
+         {
+            if (mUpdateAttributeCallback) {
+               mUpdateAttributeCallback(*this, Clusters::PressureMeasurement::Id,
+                        Clusters::PressureMeasurement::Attributes::MeasuredValue::Id, data,
+                        dataSize);
+            }
+         }
+
+   #. Implement the body of :c:func:`UpdateState` that will be called by the ``Bridge Manager`` to inform that data in Matter Data Model was changed and request propagating this information to the end device.
+      In this case, there is not any attribute supporting write operations and sending data to end device is not required, so edit :file:`simulated_pressure_sensor_data_provider.cpp` file as follows:
+
+      .. code-block:: C++
+
+         CHIP_ERROR SimulatedPressureSensorDataProvider::UpdateState(chip::ClusterId clusterId, chip::AttributeId attributeId,
+                                    uint8_t *buffer)
+         {
+            return CHIP_ERROR_UNSUPPORTED_CHIP_FEATURE;
+         }
+
+3. Include ``PressureSensorDevice`` and ``SimulatedPressureSensorDataProvider`` implementations created in a previous steps to the compilation.
+   Edit :file:`CMakeLists.txt` file, as follows:
+
+   .. code-block:: cmake
+
+      target_sources(app PRIVATE
+        src/bridged_device_types/pressure_sensor.cpp
+        src/simulated_providers/simulated_pressure_sensor_data_provider.cpp
+      )
+
+4. Provide allocators for ``PressureSensorDevice`` and ``SimulatedPressureSensorDataProvider``  objects creation.
+   The Matter Bridge application uses a factory module that creates a pair of ``Matter Bridged Device`` and ``Bridged Device Data Provider`` objects matching specific Matter device type id.
```

Comment:
```suggestion
   The Matter Bridge application uses a factory module that creates a pair of ``Matter Bridged Device`` and ``Bridged Device Data Provider`` objects matching a specific Matter device type ID.
```
On the BLE page, the factory module was named. Should it be here as well?

Reply:
Yes, I added it to be consistent.